cmd/server/pactasrv/conv/oapi_to_pacta.go

```diff
@@ -109,15 +123,65 @@ func PortfolioGroupCreateFromOAPI(pg *api.PortfolioGroupCreate, ownerID pacta.Ow
 }
 
 func auditLogActionFromOAPI(i api.AuditLogAction) (pacta.AuditLogAction, error) {
-	return pacta.ParseAuditLogAction(string(i))
+	switch i {
+	case api.AuditLogActionCREATE:
+		return pacta.AuditLogAction_Create, nil
+	case api.AuditLogActionUPDATE:
+		return pacta.AuditLogAction_Update, nil
+	case api.AuditLogActionDELETE:
+		return pacta.AuditLogAction_Delete, nil
+	case api.AuditLogActionADDTO:
+		return pacta.AuditLogAction_AddTo, nil
+	case api.AuditLogActionREMOVEFROM:
```

Comment:
nit: CANYOUADDSOMEUNDERSCORESHERE? OTHERWISEITSHARDTOREAD

ANDBELOW

(let me know if its annoying to do, I should be able to do it in one ~5s pass in a text editor)

Reply:
Went with CamelCase - couldn't figure out how to configure (globally) enum code generation.